tests/api_tests/base/test_views.py

```diff
@@ -47,6 +53,9 @@ def test_view_classes_have_minimal_set_of_permissions_classes(self):
         view_modules = [name for _, name, _ in pkgutil.iter_modules(['api'])]
         for module in view_modules:
             for name, view in inspect.getmembers(sys.modules['api.{}.views'.format(module)], inspect.isclass):
+                if getattr(view, 'permissions_exempt', False):
```

Comment:
Maybe. It's only vague in my head right now, but the general idea is: The flag is useful in this case, but we don't want someone just adding "permissions_exempt = true" to, say, the node_list_view. not that they would, and not that we wouldn't catch it if they did, but. So is there a way that makes sense to also check to see if the view isn't important? I'm leaning towards, "no, there isn't", but I thought I'd ask.